tests/hil/scripts/pytest-hil/nordicboard.py

```diff
@@ -0,0 +1,18 @@
+import time
+
+from board import Board
+from pynrfjprog import HighLevel
+
+class NordicBoard(Board):
+    def program(self, hex_file):
+        with HighLevel.API() as api:
+            with HighLevel.DebugProbe(api, int(self.snr)) as probe:
+                program_options = HighLevel.ProgramOptions(
+                    erase_action=HighLevel.EraseAction.ERASE_ALL,
+                    reset=HighLevel.ResetAction.RESET_SYSTEM,
+                    verify=HighLevel.VerifyAction.VERIFY_READ,
+                )
+
+                probe.program(hex_file, program_options=program_options)
+
+        time.sleep(3)
```

Comment:
Why do we have 3s sleep here? Maybe let's put some comment about it, why it is needed. Obviously it should state that it is a workaround, but the additional important information is what problem does it mitigate.